CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/home/HomeViewModel.kt

```diff
@@ -1,48 +1,52 @@
 package co.nimblehq.coroutine.ui.screens.home
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.viewModelScope
 import co.nimblehq.coroutine.domain.data.entity.UserEntity
-import co.nimblehq.coroutine.domain.usecase.UseCaseResult
 import co.nimblehq.coroutine.domain.usecase.GetUsersUseCase
-import co.nimblehq.coroutine.lib.Event
+import co.nimblehq.coroutine.domain.usecase.UseCaseResult
 import co.nimblehq.coroutine.ui.base.BaseViewModel
+import co.nimblehq.coroutine.ui.base.NavigationEvent
+import co.nimblehq.coroutine.ui.screens.second.SecondBundle
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 interface Output {
 
-    val users: LiveData<List<UserEntity>>
+    val users: StateFlow<List<UserEntity>>
 
-    val showError: LiveData<Event<String>>
+    fun navigateToSecond(bundle: SecondBundle)
 }
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
     private val getUsersUseCase: GetUsersUseCase
 ) : BaseViewModel(), Output {
 
-    // TODO: Re-factor LiveData to Flow and remove Event
-    private val _users = MutableLiveData<List<UserEntity>>()
-    override val users: LiveData<List<UserEntity>>
+    private val _users = MutableStateFlow<List<UserEntity>>(listOf())
```

Comment:
how about emptyList, listOf should be used for at least I item?

Reply:
Updated 🙏🏼